checks.d/sidekiq.py

```diff
@@ -0,0 +1,246 @@
+'''
+Sidekiq checks
+'''
+# stdlib
+import time
+
+# 3rd party
+import redis
+
+# project
+from checks import AgentCheck
+
+class Stats:
```

Comment:
Your class you inherit from Object.
http://stackoverflow.com/questions/7304056/why-inheriting-from-object-type